x-pack/test/apm_api_integration/tests/latency/service_apis.spec.ts

```diff
@@ -91,6 +91,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             kuery: `processor.event : "${processorEvent}"`,
             transactionType: 'request',
             latencyAggregationType: 'avg' as LatencyAggregationType,
+            useDurationSummary: false,
```

Comment:
Ideally these APIs are tested with and without the summary

```suggestion
            useDurationSummary: useDurationSummary,
```